src/xdocs/config_naming.xml

```diff
@@ -2437,6 +2437,102 @@ class MyClass {
       </subsection>
     </section>
 
+    <section name="RecordTypeParameterName">
```

Comment:
Please generate web site and share link to it to see rendering.

Reply:
https://nmancus1.github.io/site_ RecordTypeParameterName

Reply:
Please investigate and fix indentation on code blocks in html web site, please review other code samples

Reply:
Fixed in this PR, and [here](https://github.com/checkstyle/checkstyle/pull/8596), too.